x/liquidity/spec/02_state.md

```diff
@@ -50,11 +48,15 @@ type PoolBatch struct {
 }
 ```
 
-## DepositMsgState for Batch Messages
+## Batch Messages
+
+Deposit, withdrawal, or swap orders are accumulated in a liquidity pool for a pre-defined period, which can be one or more blocks in length. Orders are then added to the pool and executed at the end of the batch. The following messages are executed in batch-style. 
+
+### DepositMsgState
 
-The `DepositMsgState` state defines the state of deposit message as it is processed in the next batch or batches.
+`DepositMsgState` defines the state of deposit message as it is processed in the next batch or batches.
 
-When a user sends a MsgDeposit transaction to the network, it is accumulated in a batch. The `DepositMsgState` contains the state information about the message, if the transaction is executed, successfully matched, and if it will be deleted in the next block.
+When a user sends `MsgDepositWithinBatch` transaction to the network, it is accumulated in a batch. `DepositMsgState` contains the state information about the message; if the transaction is executed, successfully matched, and if it is to be deleted in the next block.
```

Comment:
```suggestion
When a user sends a `MsgDepositWithinBatch` transaction to the network, it is accumulated in a batch. `DepositMsgState` contains the state information about the message:

- If the transaction is executed
- If the transaction is successfully matched
- If the transaction will be deleted in the next block
```

Reply:
hi @kogisin we need to include "a" before the message for plain language or we can make "transaction" plural so the sentence reads:

> When a user sends `MsgDepositWithinBatch` transactions to the network, they are accumulated in a batch.